### ISSUE TITLE
Build with v6.18.0 fails with "Connect to gitlab.scandit.com:443 [gitlab.scandit.com/85.195.222.114] failed: connect timed out"

### ISSUE THREAD

**Issue by PatrickSpies** [Scandit/scandit-cordova-datacapture-core#4]
Trying to build an android-app with the new v6.18.0 fails with the following error:
```
> Task :app:checkDebugAarMetadata FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve androidx.core:core:1.6.+.
     Required by:
         project :app
      > Failed to list versions for androidx.core:core.
         > Unable to load Maven meta-data from https://gitlab.scandit.com/api/v4/projects/1306/packages/maven/androidx/core/core/maven-metadata.xml.
            > Could not GET 'https://gitlab.scandit.com/api/v4/projects/1306/packages/maven/androidx/core/core/maven-metadata.xml'.
               > Connect to gitlab.scandit.com:443 [gitlab.scandit.com/85.195.222.114] failed: connect timed out
   > Could not resolve com.google.android.gms:play-services-location:16.+.
     Required by:
         project :app
      > Skipped due to earlier error
      > Failed to list versions for com.google.android.gms:play-services-location.
         > Unable to load Maven meta-data from https://gitlab.scandit.com/api/v4/projects/131/packages/maven/com/google/android/gms/play-services-location/maven-metadata.xml.
            > Could not GET 'https://gitlab.scandit.com/api/v4/projects/131/packages/maven/com/google/android/gms/play-services-location/maven-metadata.xml'.
               > Connect to gitlab.scandit.com:443 [gitlab.scandit.com/85.195.222.114] failed: connect timed out
```

`src/android/build.gradle` references gitlab.scandit.com which seems to be not accessible!?

**Comment by ScanditSupport** [comment]
Hi @PatrickSpies,

Thanks for reaching out with your questions.

This issue will require further investigation on our end, can you please reach out to us directly and share a few snippets of your code on how you implemented Scandit? Could you also let me know if you have tested our samples and which one you have tested?

We will be waiting to receive your request on: https://support.scandit.com/hc/en-us/requests/new

**Comment by PatrickSpies** [comment]
Fixed mit 6.18.1

**Comment by ScanditSupport** [comment]
Hi @PatrickSpies,

Glad to hear that! Thank you for the heads up. 

In case you have more detailed questions, please feel free to reach out to us directly:
[Submit a request – Frequently Asked Questions (FAQ)](https://support.scandit.com/hc/en-us/requests/new)
